app/views/pages/additional_guidance.html.erb

```diff
@@ -9,48 +9,72 @@
-      </p>
-      <p class="govuk-!-margin-bottom-9">
-        Do not use links in hint text. While screen readers will read out the link text when describing the field, they will not tell users the text is a link
+        Use guidance if you need to:
       </p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>explain how to answer the question in more detail</li>
+        <li>provide more context</li>
+        <li>format your content - for example, with links, sub-headings or lists</li>
+      </ul>
 
-      <%= f.govuk_text_field( :page_heading, label: { size: 'm' }, hint: { text: "Use a heading that’s a statement rather than a question - for example, ‘Interview needs’"})  %>
+      <%= f.govuk_text_field( :page_heading, label: { size: 'm' }, hint: { text: "Use a heading that’s a statement rather than a question - for example, ‘Interview needs’. This will be your main page heading."})  %>
 
       <%= f.govuk_text_area( :additional_guidance_markdown,
                              label: { size: 'm' },
-                             hint: { text: "Use markdown to add formatting to your content. Formatting help can be found below the text area."},
+                             hint: { text: "Use Markdown if you need to format your guidance content. Formatting help can be found below."},
                              rows: 15)  %>
 
 
       <h2 class="govuk-heading-m">Formatting help</h2>
 
-      <p>To put a subheading in your template, use 2 hashes:</p>
+      <h3 class="govuk-heading-s">Links and URLs</h3>
+
+      <p>To add a link, use square brackets [ ] around the link text, and round brackets ( ) around the full URL. For example:</p>
+
+      <%= govuk_inset_text(text: "[Link text](https://www.gov.uk/link-text-url)") %>
+
+      <h3 class="govuk-heading-s">Second-level headings</h3>
+
+      <p>To add a second-level heading, use 2 hashtags followed by a space. For example:</p>
+
+      <%= govuk_inset_text(text: "## This is a second-level heading") %>
 
-      <%= govuk_inset_text(text: "## This is a subheading") %>
+      <h3 class="govuk-heading-s">Third-level headings</h3>
 
-      <p>To make bullet points, use asterisks:</p>
+      <p>To add a third-level heading, use 3 hashtags followed by a space. For example:</p>
+
+      <%= govuk_inset_text(text: "### This is a third-level heading") %>
+
+      <h3 class="govuk-heading-s">Bulleted lists</h3>
+
+      <p>To add bullet points, start each item with * (asterisk) or - (dash). Use one space after the asterisk or dash. For example:</p>
 
       <%= govuk_inset_text do %>
         <ul class="govuk-list">
-          <li>* point 1</li>
-          <li>* point 2</li>
-          <li>* point 3</li>
+          <li>* First bullet point</li>
+          <li>* Second bullet point</li>
+          <li>* Third bullet point 3</li>
```

Comment:
Guessing this `3` should've been removed and just got missed:
```suggestion
          <li>* Third bullet point</li>
```

Reply:
Oh yes, thanks David!